mlir/test/Dialect/Vector/invalid.mlir

```diff
@@ -525,6 +525,15 @@ func.func @test_vector.transfer_read(%arg0: memref<?x?xvector<2x3xf32>>) {
 
 // -----
 
+func.func @test_vector.transfer_read(%arg1: memref<?xindex>) -> vector<3x4xi32> {
+  %c3_i32 = arith.constant 3 : i32
```

Comment:
Let's follow the style that's already used in this file and avoid encoding the type in the variable name. `%c3_i32` -> `%c3`.